### PR TITLE
Changes for new version of libcryptsetup-rs

### PR DESCRIPTION
**Pull request by jbaublitz**
Blocked by stratis-storage/libcryptsetup-rs#58

### CODE REVIEW

**Comment by mulkieran** [comment]
Travis succeeded.

**Comment by jbaublitz** [comment]
New version of libcryptsetup-rs has been released.

File: src/engine/strat_engine/backstore/crypt.rs [hunk 1]
```diff
@@ -265,14 +265,14 @@ impl CryptInitializer {
 
         // Initialize stratis token
         log_on_failure!(
-            device.token_handle().json_set(
-                Some(STRATIS_TOKEN_ID),
+            device.token_handle().json_set(TokenInput::ReplaceToken(
```
Comment (mulkieran) [inline src/engine/strat_engine/backstore/crypt.rs:268]: This change seems correct to me, but I don't understand why it was able to compile without the change, i.e., with just the previous commit.

Comment (jbaublitz) [reply src/engine/strat_engine/backstore/crypt.rs:268]: I don't understand either. I wondered the same thing.

Comment (jbaublitz) [reply src/engine/strat_engine/backstore/crypt.rs:268]: I found the answer from looking at the git history. For some reason, libcryptsetup-rs doesn't warn us about when a branch is out of sync with master so the branch that I ran CI on didn't include the change in the PR that added `TokenInput` but master did when I released it. You can see this happening visually if you do `git log --graph --online`. However, this should now be resolved. The only potential outstanding issue is that I don't know how to get Github to check if the PR branch is out of sync with master. Do you know how to do this?

Comment (jbaublitz) [reply src/engine/strat_engine/backstore/crypt.rs:268]: After doing some digging, I found the Github settings to enforce this and this should be resolved moving forward.

Comment (mulkieran) [reply src/engine/strat_engine/backstore/crypt.rs:268]: > After doing some digging, I found the Github settings to enforce this and this should be resolved moving forward.

Ok!

**Pull request closed without merging**